debbuild

```diff
@@ -307,6 +308,7 @@ sub read_multiline_macro {
 sub load_config {
   # Load user configuration, permitting local override
   my $homedir = $ENV{HOME} // $ENV{LOGDIR} // (getpwuid($<))[7];
+  $ENV{DEB_PYTHON_INSTALL_LAYOUT} = 'deb';
```

Comment:
No. This goes in debbuild-macros, not here.

Reply:
That would also mean having to patch the python-rpm-macros package then though

Reply:
I'll give that way a shot